docs/quickstart.md

```diff
@@ -38,64 +38,113 @@ To follow along with this quickstart tutorial on your own computer, it is helpfu
 
 `mystmd` is a command line interface (CLI) that provides modern tooling for technical writing, reproducible science, and creating scientific & technical websites. To get started, install `mystmd`.
 
-:::{tip} Prerequisites
+:::{tip} Prerequisites - Install Node
 :class: dropdown
 
 You should have these programs installed:
 
-- [Node.js](https://nodejs.org) version **>=16.0.0**
+- [Node.js](https://nodejs.org) version **>=18.0.0**
```

Comment:
There is minor inconsistency with >=18 vs 16/18/20 elsewhere. We could just change it to 18/20 everywhere, gently encouraging upgrades.